gm4_double_doors/data/gm4_double_doors/templates/functions/door/north/left/toggle.mcfunction

```diff
@@ -1,17 +1,17 @@
 # Toggles the double door the player has interacted with.
 # @s = player that interacted with a door
 # at location of the lower half of the left door of the double door the player has interacted with
-# run from check_neighbours functions in gm4_double_doors:$material_name$/door/...
+# run from check_neighbours functions in gm4_double_doors:{{ material_name }}/door/...
 
 # player just closed a door
-execute if score $target_door_state gm4_double_doors_data matches 0 run place template gm4_double_doors:$material_name$/door/south/left/closed ~ ~ ~
+execute if score $target_door_state gm4_double_doors_data matches 0 run place template gm4_double_doors:{{ material_name }}/door/north/left/closed ~ ~ ~
```

Comment:
So, can you take a look at these changes to make sure something didn't go wonky? I didn't intend to replace `south` with `north` here, but it may just be git failing to properly detect a file rename properly?

Reply:
Checked the files against my original, and you did infact not change any of these directions -- git seems to have used the wrong file as a source for the changes, but the actual content is still what it is meant to be!